test/E2ETest/WebStack.QA.Test.OData/ETags/JsonETagsTests.cs

```diff
@@ -69,7 +69,6 @@ public void ModelBuilderTest()
             string expectMetadata =
                 "<EntitySet Name=\"ETagsCustomers\" EntityType=\"WebStack.QA.Test.OData.ETags.ETagsCustomer\">\r\n" +
                 "          <NavigationPropertyBinding Path=\"RelatedCustomer\" Target=\"ETagsCustomers\" />\r\n" +
-                "          <NavigationPropertyBinding Path=\"ContainedCustomer\" Target=\"ContainedCustomer\" />\r\n" +
```

Comment:
why have this one?

Reply:
It is from the tip of the master branch. Should disappear after rebase. Thanks!